ldap/servers/slapd/back-ldbm/back-ldbm.h

```diff
@@ -502,7 +497,7 @@ struct attrinfo
     DataList *ai_idlistinfo;             /* fine grained id list */
 };
 
-#define MAXDBCACHE 20
+#define MAXdbi_cursor_tACHE 20
```

Comment:
case?

Reply:
in fact the #define is no more used and should be removed ...
Seeing your remark I searched for "dbi_cursor_t[^ ]" and "[a-zA-Z0-9_]dbi_cursor_t" and the good new is that there is no other unwanted substitution.